fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -3519,6 +3528,37 @@ private Table createOlapTable(Database db, CreateTableStmt stmt, boolean isResto
             throw new DdlException(e.getMessage());
         }
 
+        // check all dynamic properties exist
+        if (properties != null) {
```

Comment:
would better move this section code to `DynamicPartitionUtils`. Catalog class is already  too big.

Reply:
> would better move this section code to `DynamicPartitionUtils`. Catalog class is already too big.

Ok, i will remove it